interpreter/cling/lib/MetaProcessor/MetaProcessor.cpp

```diff
@@ -473,7 +477,15 @@ namespace cling {
     if (topmost)
       m_TopExecutingFile = m_CurrentlyExecutingFile;
 
-    content.insert(0, "#line 2 \"" + filename.str() + "\" \n");
+    std::string path(filename.str());
+#ifdef LLVM_ON_WIN32
```

Comment:
Could we add a cling test exposing this issue?

Reply:
Well, almost all the ROOT tests should fail with something like:
454: input_line_11:2:13: error: \U used with no following hex digits
454:  #line 2 "C:\Users\bellenot\build\master\runtutorials\rootlogon.C"